### DISCUSSION TITLE
WebView

### DISCUSSION THREAD

**Discussion by pychans** [flet-dev/flet#4382] [General]
When will it be updated? I have been waiting for the client version of webview for a year and I am looking forward to it very much.

**Comment by pychans** [comment]
Is there any plan for this?

**Comment by ndonkoHenri** [comment]
We have WebView control: https://docs.flet.dev/webview/
But it doesnt support Linux and Windows atm.

**Comment by pychans** [comment]
Is there any plan for  Linux and Windows?

**Reply by hschneider** [reply]
Having this on all platforms would be awesome. 

But the problem here is the availability of the wrapped Flutter control, which is limited to Android, iOS and macOS: https://pub.dev/packages/webview_flutter

Maybe this would be a better solution:
https://pub.dev/packages/flutter_inappwebview
But there needs to be a Flet wrapper first.
